guides/client/dom-patching.md

```diff
@@ -23,6 +23,10 @@ with another JS library. The "append" and "prepend" feature is often
 used with "Temporary assigns" to work with large amounts of data. Let's
 learn more.
 
+*Note*: "ignore" behaviour ignores changes to content of the element.
+If your JS library changes attributes of the element (e.g. adds classes)
+, you should add another element inside "ignore"d element and use that one.
+
```

Comment:
I think it would be even more helpful to the reader to move this from a "note" as a third paragraph, to being an additional sentence (and not a "note"), after the existing "ignore" discussion in the preceding paragraph.  Then move the "append" and "prepend" discussion to a new paragraph.

```
The "ignore" behaviour is frequently used when you need to integrate with another JS library. <YOUR ADDITIONS>

The "append" and "prepend" feature is often ... 
```

This would have two additional benefits:

1.  This change seems important, so the reader is more likely to read it.
2.  It would preserve "Let's learn more." leading into the next section on temporary assigns.

I think it would also be helpful to describe why a container element should be used, because it seems like a work-around.. but for what problem?  Is this doc addition pointing out that "ignore" does *not* ignore changes to the element itself, but only to the _content_?  (i.e., LV will still change the element itself, and hence JS will compete with LV?)